packages/wdio-sync/package.json

```diff
@@ -17,6 +17,7 @@
     "test:eslint": "eslint src test",
     "test:unit": "jest"
   },
+  "types": "./webdriverio-sync.d.ts",
```

Comment:
According to webdriverio-generate-typings.js this file needs to be just called `webdriverio.d.ts`

Reply:
we now have 
webdriverio.d.ts in both @wdio/sync and webdriverio
webdriverio-sync.d.ts in @wdio/sync
webdriverio-async.d.ts in webdriverio

Reply:
ah ok, makes sense. Could you update the docs to reflect this?

Reply:
yes, I have some tasks to do besides of docs:
* addCommand should not be a promise
* webDriver commands should return promise
* some command should be handled separately like WebDriver.newSession, remote, multiremote, etc.
* update docs

Reply:
@christian-bromann TypeScript docs updaed